### ISSUE TITLE
`Permalink` and `Portal link` are different than stock intel

### ISSUE THREAD

**Issue by nexushoratio** [IITC-CE/ingress-intel-total-conversion#684]
Stock intel generates URLs like this:
https://intel.ingress.com/intel?ll=37.423521,-122.089649&z=17
https://intel.ingress.com/intel?ll=37.423521,-122.089649&z=17&pll=37.423521,-122.089649
That is WITH `/intel` in the path name.

IITC generates URLs like this:
https://intel.ingress.com/?ll=37.423521,-122.089649&z=17
https://intel.ingress.com/?pll=37.423521,-122.089649
This is, with OUT the `/intel` portion.

Also, the game's own deep links include `/intel`.

While stock works without the `/intel` portion, it seems like a good way to track those using IITC.

Also, at least one Android helper app, [Portal Intel URL to GPS](https://play.google.com/store/apps/details?id=com.marcosdiez.ingressurltogps) does not recognize the links without `/intel` in them.  I filed https://github.com/marcosdiez/IngressIntelToGps/issues/4 for that.  (Though considering it was last updated Nov 15, 2018, and the fact that the author has a more generic app that handles IITC generated URLs, I'm not sure it would ever be updated.)

**Comment by xscreach** [comment]
Imho this is only result of niantic ability to do things properly.

They changed the intel URL from www.ingress.com/intell to intell.ingress.com several years ago and failed to reflect this everywhere.
They rather made intell work both with and without the /intell part.

So nothing to do on IITC side 🤷

**Comment by nexushoratio** [comment]
Stock's URL includes the string `intel` twice.  In the host and the path.  The old host name doesn't come into play.

Using `www.ingress.com/intel` will redirect to `intel.ingress.com/intel` NOT `intel.ingress.com/`

IITC has always been about trying to be indistinguishable from stock intel in order to reduce chance of detection and not abuse the servers (and in some cases, even more efficient than stock).

This seems like an unnecessary difference.

**Comment by DanielOndiordna** [comment]
I agree with nexushoratio: only on Desktop IITC and IITC Button there is a different Portal link and Permalink.
IITC mobile for Android and the iOS version both format the links the same as stock intel does.
Stock intel portal links also include the map location.

What I understand is that the function window.makePermalink is used to create the Permalink and Portal link.
On mobile, this function is not used and replaced by Share portal which calls window.app.intentPosLink
intentPosLink creates a link including "/intel" and map location for portal links, just like stock intel links.

As long as the IITC core function is not changed, I have published a plugin on my website that will fix the makePermalink function, for those in need of stock intel links on Desktop IITC:
https://softspot.nl/ingress/plugindetails/iitc-plugin-fixdesktopportallinks
